dev-server/templates/client-config.js.mustache

```diff
@@ -5,6 +5,14 @@
       // showHighlights: 'always',
       // theme: 'clean',
 
+      // Example branding:
```

Comment:
This is just for increased dev ergonomics

Reply:
I didn't know about this mechanism to inject client config. It is useful.

Reply:
This is unrelated to this PR, but could you please add a new line on top of `Example service config`?

Reply:
Done!